src/org/exist/util/VirtualTempFile.java

```diff
@@ -43,6 +43,7 @@
  * @author jmfernandez
  *
  */
+@Deprecated
 public class VirtualTempFile
```

Comment:
I just added some comments on that in the javadoc and rebased it for you. Hopefully we are all good now.